ietf/submit/tests.py

```diff
@@ -237,7 +237,7 @@ def create_and_post_submission(self, name, rev, author, group=None, formats=("tx
                     self.assertTrue(os.path.exists(os.path.join(self.staging_dir, "%s-%s.%s" % (name, rev, 'html'))))
         return r
 
-    def do_submission(self, name, rev, group=None, formats: Tuple[str, ...]=("txt",), author=None, base_filename=None, ascii=True):
+    def do_submission(self, name, rev, group=None, formats: Tuple[str, ...]=("txt",), author=None, email=None, base_filename=None, ascii=True):
```

Comment:
I should probably have caught this the first time around, but I don't think this `email` parameter is used either. Worries me that the test is not testing what it means to test.